dags/main_summary.py

```diff
@@ -43,6 +45,94 @@
+main_summary_dataproc_bucket = "gs://moz-fx-data-derived-datasets-parquet-tmp"
+main_ping_bigquery_export_prefix = main_summary_dataproc_bucket + "/export"
+main_ping_bigquery_export_dest = main_ping_bigquery_export_prefix + "/submission_date={{ds}}/document_namespace=telemetry/document_type=main/document_version=4/*.avro"  # noqa
+main_ping_bigquery_export = gke_command(
+    task_id="main_ping_bigquery_extract",
+    command=[
+        "bq",
+        "extract",
+        "--destination_format=AVRO",
+        "moz-fx-data-shared-prod.payload_bytes_decoded.telemetry_telemetry__main_v4${{ds_no_dash}}",
+        main_ping_bigquery_export_dest,
+    ],
+    docker_image=gcloud_docker_image,
+)
+
+main_summary_dataproc = SubDagOperator(
+    subdag=moz_dataproc_jar_runner(
+        parent_dag_name="main_summary",
+        dag_name="main_summary_dataproc",
+        default_args=default_args,
+        cluster_name="main-summary-{{ds}}",
+        image_version="1.3",
+        idle_delete_ttl="600",
+        worker_machine_type="n1-standard-8",
+        num_preemptible_workers=40,
+        optional_components=[],
+        install_component_gateway=False,
+        jar_urls=[
+            "s3a://net-mozaws-data-us-west-2-ops-ci-artifacts/mozilla/telemetry-batch-view/master/telemetry-batch-view.jar",
+        ],
+        main_class="com.mozilla.telemetry.views.MainSummaryView",
+        jar_args=[
+            "--from={{ds_nodash}}",
+            "--to={{ds_nodash}}",
+            "--bucket=" + main_summary_dataproc_bucket,
+            "--export-path=" + main_ping_bigquery_export_prefix,
+        ],
+        artifact_bucket=None,
+        gcp_conn_id="google_cloud_airflow_dataproc",
+    ),
+    task_id="main_summary_dataproc",
+    dag=dag,
+)
+
+main_summary_dataproc_bigquery_load = SubDagOperator(
+    subdag=load_to_bigquery(
+        parent_dag_name=dag.dag_id,
+        dag_name="main_summary_dataproc_bigquery_load",
+        default_args=default_args,
+        dataset_gcs_bucket=main_summary_dataproc_bucket.replace("gs://", ""),
+        dataset="main_summary",
+        dataset_version="v4",
+        gke_cluster_name="bq-load-gke-1",
+        bigquery_dataset="backfill",
+        cluster_by=["sample_id"],
+        drop=["submission_date"],
+        rename={"submission_date_s3": "submission_date"},
+        replace=["SAFE_CAST(sample_id AS INT64) AS sample_id"],
+    ),
+    task_id="main_summary_dataproc_bigquery_load",
+    dag=dag,
+)
+
+main_ping_bigquery_export_delete = gke_command(
+    task_id="main_ping_bigquery_export_delete",
+    command=[
+        "gsutil",
+        "-m",
+        "rm",
+        main_ping_bigquery_export_dest,
+    ],
+    docker_image=gcloud_docker_image,
+)
+
+main_summary_dataproc_s3_copy = gke_command(
+    task_id="main_summary_dataproc_s3_copy",
+    command=[
+        "gsutil",
+        "-m",
+        "rsync",
+        main_summary_dataproc_bucket + "/main_summary/v4/submission_date_s3={{ds_no_dash}}",
```

Comment:
does this work? or should it be {{ ds_nodash }}

Reply:
>_< should be `ds_nodash`, fixing...